modules/statics/src/coins.ts

```diff
@@ -219,6 +219,7 @@ const SUI_FEATURES = [
   CoinFeature.TSS_COLD,
   CoinFeature.STAKING,
   CoinFeature.BULK_TRANSACTION,
+  CoinFeature.BULK_STAKING_TRANSACTION,
```

Comment:
why enabling it for sui also ?

Reply:
It is already enabled for SUI in the UI by hardcoding. Creating this coin feature to remove the hardcoding thats why enabling it for SUI also